src-tauri/src/northstar/mod.rs

```diff
@@ -112,8 +113,17 @@ pub fn launch_northstar(
             || matches!(game_install.install_type, InstallType::UNKNOWN))
     {
         let ns_exe_path = format!("{}/NorthstarLauncher.exe", game_install.game_path);
+
+        let mut args = vec!["/C", "start", "", &ns_exe_path];
+        // We cannot add the params directly because of limitations with cmd.exe
+        // https://stackoverflow.com/questions/9964865/c-system-not-working-when-there-are-spaces-in-two-different-parameters/9965141#9965141
+
+        let launch_parameters = launch_parameters.unwrap_or_default();
+        let ns_params: Vec<&str> = launch_parameters.split_whitespace().collect();
+        dbg!(ns_params.clone());
+        args.extend(ns_params);
         let _output = std::process::Command::new("C:\\Windows\\System32\\cmd.exe")
-            .args(["/C", "start", "", &ns_exe_path])
+            .args(args)
```

Comment:
cc @Jan200101 cause this will 100% merge conflict with #444 so we should probably figure out how to best merge it ^^"

I.e. turn profile command into launch arg and then pass it or keep profile arg in `GameInstall` struct and then combine inside the Rust code.
(second one probably makes more sense)

Reply:
A lot of APIs rely on having profile information available, that would require adapting, which is why I made the profile part of `GameInstall`.

Reply:
Yeah, profile should stay in `GameInstall` so second option it is then ^^